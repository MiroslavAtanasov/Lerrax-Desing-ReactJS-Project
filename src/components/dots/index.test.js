import React from 'react'
import renderer from 'react-test-renderer'
import Dots from '.'
import TestingEnvironment from '../../test-utils/router'


describe('Dots component', () => {
    it('should render Dots component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Dots />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})