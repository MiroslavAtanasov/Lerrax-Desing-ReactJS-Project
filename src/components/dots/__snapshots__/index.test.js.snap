// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dots component should render Dots component 1`] = `
<span
  className="dot"
/>
`;
